app/views/modal_partials/_two_factor_registration_complete.html.slim

```diff
@@ -0,0 +1,10 @@
+.col.col-center.col-xs-12.col-sm-8.col-md-8.col-lg-9.text-center
+  = image_tag "2fa-good@2x.png", width: 60, height: 72
```

Comment:
Are we not using the SVG then? This PR includes both SVG and PNG icons...

Reply:
I'm going to swap this PR over to use an inline SVG 👍